gulpfile.js

```diff
@@ -1,11 +1,85 @@
-var gulp = require('gulp');
-var uglify = require('gulp-uglify');
-var rename = require('gulp-rename');
 
-gulp.task('build', function() {
- gulp.src('diver.js')
-     .pipe(gulp.dest('dist/'))
-     .pipe(rename('diver.min.js')) 
-     .pipe(uglify())
-     .pipe(gulp.dest('dist'))
+var
```

Comment:
Since the issue says just about the gulp watch. Add it only for this commit/pull request.